SYCL/DeprecatedFeatures/buffer_interop.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: opencl, opencl_icd
 
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %opencl_lib
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -D__SYCL_INTERNAL_API -o %t.out %opencl_lib
```

Comment:
Which APIs necessitate this macro in the tests?

Reply:
We remove some APIs by wrapping them with `#ifdef __SYCL_INTERNAL_API`, in this particular case old OpenCL-interop APIs are removed (since tests and impls are in different repos, we have to commit test changes first).